app/build.gradle

```diff
@@ -0,0 +1,27 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 23
+    buildToolsVersion "23.0.1"
+
+    defaultConfig {
+        applicationId "com.example.edwinmperazaduran.gridimagesearch"
+        minSdkVersion 14
```

Comment:
@edwinperaza Nowadays you should target 16 and above. 14 is pretty old and not worth the extra work to support it